### PR TITLE
feat: support the Map state

### PR DESCRIPTION
**Pull request by theburningmonk**
official documentation [here](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-map-state.html)

### CODE REVIEW

File: src/schemas/map.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    "ItemsPath": {
+      "type": ["string", "null"]
+    },
+    "MaxConcurrency": {
+      "type": "number",
+      "minimum": 0
+    },    
+    "Iterator": {
+      "$ref": "http://asl-validator.cloud/state-machine#"
+    },
```
Comment (ChristopheBougere) [inline src/schemas/map.json]: I think we are missing the `Parameters` property which is kinda [hidden in the docs](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-map-state.html#map-state-example-params), but its usage is confirmed in [ASL specs](https://states-language.net/spec.html#state-type-table):
image

Comment (theburningmonk) [reply src/schemas/map.json]: well caught!

Comment (theburningmonk) [reply src/schemas/map.json]: added!

**Comment by ChristopheBougere** [comment]
Released in [asl-validator@1.5.0](https://www.npmjs.com/package/asl-validator/v/1.5.0), thanks for contributing!

**Comment by theburningmonk** [comment]
👍 great, thanks! I'll update the serverless-step-functions plugin as well

**Pull request merged**